crates/viewer/re_view_spatial/src/transform_cache.rs

```diff
@@ -201,55 +256,104 @@ impl TransformCacheStoreSubscriber {
     ///
     /// Returns `None` if the timeline doesn't have any transforms at all.
     #[inline]
-    pub fn transforms_per_timeline(
-        &self,
-        timeline: Timeline,
-    ) -> Option<&CachedTransformsPerTimeline> {
-        self.per_timeline.get(&timeline)
+    pub fn transforms_for_timeline(&self, timeline: Timeline) -> &CachedTransformsForTimeline {
+        self.per_timeline
+            .get(&timeline)
+            .unwrap_or(&self.static_timeline)
     }
 
     /// Makes sure the transform cache is up to date with the latest data.
     ///
     /// This needs to be called once per frame prior to any transform propagation.
     /// (which is done by [`crate::contexts::TransformTreeContext`])
+    // TODO(andreas): easy optimization: apply only updates for a single timeline at a time.
     pub fn apply_all_updates(&mut self, entity_db: &EntityDb) {
         re_tracing::profile_function!();
 
+        // Update static transforms.
+        for invalidated_transform in self.static_timeline.invalidated_transforms.drain(..) {
+            let InvalidatedTransforms {
+                entity_path,
+                aspects,
+                ..
+            } = invalidated_transform;
+
+            let static_transforms = self
+                .static_timeline
+                .per_entity
+                .entry(entity_path.hash())
+                // There have never been any static or non-static transforms for this entity, therefore there's nothing to pass
+                // into that entity for static transforms.
+                .or_insert_with(|| TransformsForEntity::new(None, None));
+
+            // Technically this doesn't query static components but rather just what's at the beginning of the tick timeline,
+            // but it's the most convenient way to the data we want.
+            let query = LatestAtQuery::new(Timeline::log_tick(), TimeInt::MIN);
```

Comment:
Could this break if the user decided to somehow `rr.disable_timeline("log_tick")`?

Reply:
no, if anything it would make this more accurate since then we're sure that this can only yield static components. As is, this could actually give us non-static results. I think what that means is that I shouldn't use `log_tick` but something more made up...

Reply:
Makes sense!